### PR TITLE
faq: add way how to view tidb configuration file in kubernetes

### PR DESCRIPTION
**Pull request by xiaojingchen**
### What is changed, added or deleted? 

add way how to view tidb configuration file in kubernetes.

### Checklist 

- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by aylei** [comment]
Please fix the CI

File: dev/faq/tidb-in-kubernetes.md [hunk 1]
```diff
@@ -50,3 +50,31 @@ TiDB Operator 尚不支持自动编排 TiSpark。
 假如要为 TiDB in Kubernetes 添加 TiSpark 组件，你需要在**同一个** Kubernetes 集群中自行维护 Spark，确保 Spark 能够访问到 PD 和 TiKV 实例的 IP 与端口，并为 Spark 安装 TiSpark 插件，TiSpark 插件的安装方式可以参考 [TiSpark](/tispark/tispark-user-guide_v1.x.md#已有-Spark-集群的部署方式)。
 
 在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)。
+
+## 如何查看 TiDB 集群配置
+
+如果需要查看当前集群的 PD/TIKV/TIDB 组件的配置信息，可以执行下列命令
```
Comment (weekface) [inline dev/faq/tidb-in-kubernetes.md]:
```suggestion
如果需要查看当前集群的 PD/TiKV/TiDB 组件的配置信息，可以执行下列命令
```

File: v3.0/faq/tidb-in-kubernetes.md [hunk 2]
```diff
@@ -24,29 +24,57 @@ category: FAQ
...
 TiDB Operator 尚不支持自动编排 TiSpark。
 
 假如要为 TiDB in Kubernetes 添加 TiSpark 组件，你需要在**同一个** Kubernetes 集群中自行维护 Spark，确保 Spark 能够访问到 PD 和 TiKV 实例的 IP 与端口，并为 Spark 安装 TiSpark 插件，TiSpark 插件的安装方式可以参考 [TiSpark](/tispark/tispark-user-guide_v1.x.md#已有-Spark-集群的部署方式)。
 
-在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)
+在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)。
+
+## 如何查看 TiDB 集群配置
+
+如果需要查看当前集群的 PD/TIKV/TIDB 组件的配置信息，可以执行下列命令
```
Comment (weekface) [inline v3.0/faq/tidb-in-kubernetes.md]: ditto

File: dev/faq/tidb-in-kubernetes.md [hunk 3]
```diff
@@ -50,3 +50,31 @@ TiDB Operator 尚不支持自动编排 TiSpark。
 假如要为 TiDB in Kubernetes 添加 TiSpark 组件，你需要在**同一个** Kubernetes 集群中自行维护 Spark，确保 Spark 能够访问到 PD 和 TiKV 实例的 IP 与端口，并为 Spark 安装 TiSpark 插件，TiSpark 插件的安装方式可以参考 [TiSpark](/tispark/tispark-user-guide_v1.x.md#已有-Spark-集群的部署方式)。
 
 在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)。
+
+## 如何查看 TiDB 集群配置
+
+如果需要查看当前集群的 PD/TiKV/TiDB 组件的配置信息，可以执行下列命令
```
Comment (aylei) [inline dev/faq/tidb-in-kubernetes.md]:
```suggestion
如果需要查看当前集群的 PD、TiKV、TiDB 组件的配置信息，可以执行下列命令：
```

File: dev/faq/tidb-in-kubernetes.md [hunk 4]
```diff
@@ -50,3 +50,31 @@ TiDB Operator 尚不支持自动编排 TiSpark。
...
+
+查看 PD 配置文件
```
Comment (aylei) [inline dev/faq/tidb-in-kubernetes.md]:
```suggestion
* 查看 PD 配置文件：
```

最好用 ul，下同。

Comment (aylei) [reply dev/faq/tidb-in-kubernetes.md]: 用了 ul 之后注意缩进

File: dev/faq/tidb-in-kubernetes.md [hunk 5]
```diff
@@ -50,3 +50,31 @@ TiDB Operator 尚不支持自动编排 TiSpark。
 假如要为 TiDB in Kubernetes 添加 TiSpark 组件，你需要在**同一个** Kubernetes 集群中自行维护 Spark，确保 Spark 能够访问到 PD 和 TiKV 实例的 IP 与端口，并为 Spark 安装 TiSpark 插件，TiSpark 插件的安装方式可以参考 [TiSpark](/tispark/tispark-user-guide_v1.x.md#已有-Spark-集群的部署方式)。
 
 在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)。
+
+## 如何查看 TiDB 集群配置
```
Comment (aylei) [inline dev/faq/tidb-in-kubernetes.md]:
```suggestion
## 如何查看 TiDB 集群配置？
```

File: v3.0/faq/tidb-in-kubernetes.md [hunk 6]
```diff
@@ -24,29 +24,57 @@ category: FAQ
...
 ## TiDB Operator 支持 TiSpark 吗？
 
 TiDB Operator 尚不支持自动编排 TiSpark。
 
 假如要为 TiDB in Kubernetes 添加 TiSpark 组件，你需要在**同一个** Kubernetes 集群中自行维护 Spark，确保 Spark 能够访问到 PD 和 TiKV 实例的 IP 与端口，并为 Spark 安装 TiSpark 插件，TiSpark 插件的安装方式可以参考 [TiSpark](/tispark/tispark-user-guide_v1.x.md#已有-Spark-集群的部署方式)。
 
-在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)
+在 Kubernetes 上维护 Spark 可以参考 Spark 的官方文档：[Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)。
+
+## 如何查看 TiDB 集群配置
```
Comment (aylei) [inline v3.0/faq/tidb-in-kubernetes.md]:
```suggestion
## 如何查看 TiDB 集群配置？
```

File: v3.0/faq/tidb-in-kubernetes.md [hunk 7]
```diff
@@ -24,29 +24,57 @@ category: FAQ
 
 ## TiDB 相关组件可以配置 HPA 或 VPA 么？
 
-TiDB 集群目前还不支持 HPA (自动水平扩缩容）和 VPA（自动垂直扩缩容），因为对于数据库这种有状态应用而言，实现自动扩缩容还是有很大的难度的，不能单单以 CPU 和 memory 监控数据来简单地实现。
+TiDB 集群目前还不支持 HPA（Horizontal Pod Autoscaling，自动水平扩缩容）和 VPA（Vertical Pod Autoscaling，自动垂直扩缩容），因为对于数据库这种有状态应用而言，实现自动扩缩容难度较大，不能单以 CPU 和 memory 监控数据来简单地实现。
```
Comment (TomShawn) [inline v3.0/faq/tidb-in-kubernetes.md]:
```suggestion
TiDB 集群目前还不支持 HPA（Horizontal Pod Autoscaling，自动水平扩缩容）和 VPA（Vertical Pod Autoscaling，自动垂直扩缩容），因为对于数据库这种有状态应用而言，实现自动扩缩容难度较大，不能仅通过 CPU 和 memory 监控数据来简单地实现。
```

File: v3.0/faq/tidb-in-kubernetes.md [hunk 8]
```diff
@@ -24,29 +24,57 @@ category: FAQ
...
 
 ## 使用 TiDB Operator 编排 TiDB 集群时，有什么场景需要人工介入操作吗？
 
-不考虑 Kubernetes 集群本身运维，TiDB Operator 存在以下可能需要人工介入的场景：
+如果不考虑 Kubernetes 集群本身的运维，TiDB Operator 存在以下可能需要人工介入的场景：
 
-* TiKV 自动故障转移之后的集群调整，参考 [自动故障转移](/how-to/maintain/tidb-in-kubernetes/auto-failover.md)；
-* 维护或下线指定的 Kubernetes 节点，参考 [维护节点](/how-to/maintain/tidb-in-kubernetes/tidb-node.md) 。
+* TiKV 自动故障转移之后的集群调整，参考[自动故障转移](/how-to/maintain/tidb-in-kubernetes/auto-failover.md)；
+* 维护或下线指定的 Kubernetes 节点，参考[维护节点](/how-to/maintain/tidb-in-kubernetes/tidb-node.md) 。
```
Comment (TomShawn) [inline v3.0/faq/tidb-in-kubernetes.md]:
```suggestion
* 维护或下线指定的 Kubernetes 节点，参考[维护节点](/how-to/maintain/tidb-in-kubernetes/tidb-node.md)。
```

File: v3.0/faq/tidb-in-kubernetes.md [hunk 9]
```diff
@@ -24,29 +24,57 @@ category: FAQ
...
 
 ## 在公有云上使用 TiDB Operator 编排 TiDB 集群时，推荐的部署拓扑是怎样的？
 
-首先，为了实现高可用和数据安全，我们推荐生产环境的 TiDB 集群至少部署在三个可用区(Available Zone) 中。
+首先，为了实现高可用和数据安全，我们推荐生产环境的 TiDB 集群至少部署在三个可用区 (Available Zone) 中。
```
Comment (TomShawn) [inline v3.0/faq/tidb-in-kubernetes.md]:
```suggestion
首先，为了实现高可用和数据安全，我们在推荐生产环境下的 TiDB 集群中至少部署在三个可用区 (Available Zone)。
```

**Pull request merged**